content/apidocs-mxsdk/apidocs/old-version/team-server-api.md

```diff
@@ -0,0 +1,210 @@
+---
+title: "Team Server API"
+category: "API Documentation"
+menu_order: 65
+---
+
+{{% alert type="warning" %}}
+The API described on this page will be deprecated in the future. Please use the [new team server API](../team-server-api.md).
+{{% /alert %}}
+
+## 1 Introduction
+
+The team server API allows you to retrieve the information (branches, revisions) of application models stored in our team server. You always access an application model via the context of an application (see the Deploy API for more information about retrieving applications and application identifiers).
+The image below provides a domain model representation of the concepts discussed below and how these are related:
+![](attachments/131076/425989.png)
```

Comment:
We use an absolute path for cross-reference, for example, here it's better to change it to:(/apidocs-mxsdk/apidocs/team-server-api).